content/tracing/setup/php.md

```diff
@@ -8,75 +8,100 @@ further_reading:
+deb -i ddtrace.deb
+
+# using tar.gz archive (Other distributions using libc6)
+tar -xf ddtrace.tar.gz -C /
+/opt/datadog-php/bin/post_install.sh
+```
+
+Finally, install DataDog tracing library using composer:
+
+```bash
+composer config minimum-stability beta # required to install opentracing 1.0.0-beta5
+composer require opentracing/opentracing
+composer require datadog/dd-trace
+```
+
+**Note:** You can [install the DataDog library without changing minimum-stability][6]
 
 ## Compatibility
 
 PHP APM includes support for the following PHP versions:
 
 | Version | Support type |
 | -----   | ------------ |
-| 5.6.x   | Alpha        |
-| 7.0.x   | Alpha        |
-| 7.1.x   | Alpha        |
-| 7.2.x   | Alpha        |
+| 7.0.x   | Public Beta  |
+| 7.1.x   | Public Beta  |
+| 7.2.x   | Public Beta  |
+| 5.6.x   | Coming Soon  |
 
-### Integrations
+## Automatic Instrumentation
+
+Automatic instrumentation uses `ddtrace` extension to modify runtime and inject custom PHP code around specific methods. When [tracing is enabled][5] the PHP tracer is able to automatically instrument all supported libraries out of the box.
 
-APM PHP support is in alpha and provides limited automatic instrumentation. Planned beta support will provide automatic instrumentation for popular frameworks and libraries. See some of these listed below.
-Don't see your desired frameworks or libraries? Let Datadog know more about your needs through [this survey][1].
+Automatic instrumentation captures:
+
+* Method execution time.
+* Relevant trace data such as URL and status response codes for web requests or SQL query for database access
+* Unhandled exceptions, including stacktraces if available.
+* A total count of traces (e.g. web requests) flowing through the system.
+
+### Integrations
 
 #### Framework Compatibility
 
 | Module         | Versions    | Support Type       |
 | :-----------   | :---------- | :----------------- |
-| Laravel        | 5.x         | Alpha              |
-| Symfony        | 3.x         | Alpha              |
-| Magento        | 2.x         | Coming Soon [beta] |
-| Zend Framework | 3.x         | Coming Soon [beta] |
-| CakePHP        | 3.x         | Coming Soon [beta] |
-| Drupal         | 7.x         | Coming Soon [beta] |
-| Wordpress      | 4.x         | Coming Soon [beta] |
-| Slim           | 3.x         | Coming Soon [beta] |
-
-Don't see your desired web frameworks? Let Datadog know more about your needs through [this survey][1].
-
-[1]: https://goo.gl/forms/rKjH2J6nJ585KXri2
+| Laravel        | 5.x         | Public Beta        |
+| Symfony        | 3.x         | Public Beta        |
+| Magento        | 2.x         | Coming Soon        |
+| Zend Framework | 3.x         | Coming Soon        |
+| CakePHP        | 3.x         | Coming Soon        |
+| Drupal         | 7.x         | Coming Soon        |
+| Wordpress      | 4.x         | Coming Soon        |
+| Slim           | 3.x         | Coming Soon        |
```

Comment:
Can we add Yii to this list? Since many customers seem to be interested in it.

Reply:
See usage stats from survey here: https://docs.google.com/document/d/1VOcw02UUoivEOD1M_hB5YKZdnkWrYWEHBXfJzqJHi-8/edit#heading=h.nft3ecno89ze